idb/postgres/postgres.go

```diff
@@ -1518,10 +1519,17 @@ var statusStrings = []string{"Offline", "Online", "NotParticipating"}
 
 const offlineStatusIdx = 0
 
-func (db *PostgresIndexerDb) yieldAccountsThread(ctx context.Context, opts idb.AccountQueryOptions, rows *sql.Rows, tx *sql.Tx, blockheader types.Block, out chan<- idb.AccountRow) {
-	defer tx.Rollback()
+func (db *PostgresIndexerDb) yieldAccountsThread(req *getAccountsRequest) {
+	defer req.tx.Rollback()
 	count := uint64(0)
-	for rows.Next() {
+	defer func() {
+		end := time.Now()
+		dt := end.Sub(req.start)
+		if dt > (1 * time.Second) {
+			log.Printf("long query %fs: %s", dt.Seconds(), req.query)
```

Comment:
I think this should be at least a `log.Warn`. Printf ends up at the `Info` level.

Reply:
Did you fork master? This is missing some commits for develop. I renamed `postgres.PostgresIndexerDb` to `postgres.IndexerDb` to appease the linter. Also it should now have a logger instance, so you can use `db.log.Printf` instead of this StandardLogger trick.